bndtools.core/src/bndtools/editor/common/BndEditorPart.java

```diff
@@ -17,6 +17,7 @@
 
     protected BndEditModel model;
 
+    private boolean committing = false;
```

Comment:
I don't think its safe to assume that it would only be modified from UI Thread, I can switch it to AtomicBoolean.